packages-exp/firebase-exp/firestore/lite/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "firebase-exp/firestore/lite",
+  "main": "dist/index.cjs.js",
+  "browser": "dist/index.esm.js",
+  "module": "dist/index.esm.js",
+  "typings": "dist/firestore/lite/index.d.ts"
```

Comment:
Are these values used at all? Looks like the release script replaces them. Might make sense to use dummy values if that is indeed the case.

Reply:
Yes. They are used when you do `import {...} from 'firebase/firestore/lite'`